test/test_dht.cpp

```diff
@@ -1069,7 +1069,7 @@ TORRENT_TEST(dht)
 		printf("num_active_buckets: %d\n", tbl.num_active_buckets());
 		TEST_EQUAL(tbl.num_active_buckets(), 6);
    
-#if defined TORRENT_DHT_VERBOSE_LOGGING || defined TORRENT_DEBUG
```

Comment:
So then you meant `&&` instead of `||`? Because my patch preserves the current logic, nothing sets `TORRENT_DHT_VERBOSE_LOGGING` anymore these days so it always depends on `TORRENT_DEBUG`.